crates/pop-cli/src/commands/call/contract.rs

```diff
@@ -842,7 +894,7 @@ mod tests {
 			dry_run: false,
 			execute: false,
 			dev_mode: false,
-		}.configure(&mut cli, false).await, Err(message) if message.to_string().contains("Unable to fetch contract metadata: Failed to find any contract artifacts in target directory.")));
```

Comment:
Now the `ensure_contract_built` has been moved the error: `Failed to find any contract artifacts in target directory.` would never reach. But still a valid test to see what happens if invalid artifacts in the target folder